commands/build_test.go

```diff
@@ -0,0 +1,134 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	ocispecs "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+var repoDir string
+
+func setupTest(tb testing.TB) func(tb testing.TB) {
+	repoDir = tb.TempDir()
+	cmd := exec.Command("git", "init")
+	cmd.Dir = repoDir
+	err := cmd.Run()
+	if err != nil {
+		tb.Errorf("failed to init git repo: %v", err)
```

Comment:
btw, we do use `stretchr/testify` in the tests. It is not really a requirement if the tests are simple and don't require additional library but FYI in case you didn't notice.